frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -136,7 +144,7 @@ export default ({
     currentUserAttr () {
       return {
         ...this.$store.getters.ourUserIdentityContract.attributes,
-        id: currentUserId
+        id: this.$store.state.loggedIn.identityContractID
```

Comment:
Could use a getter here (`ourIdentityContractId` from `state.js`).

Reply:
Okay. I used it instead of `loggedIn.identityContractID`